### ISSUE TITLE
Pretrained model in dropbox

### ISSUE THREAD

**Issue by daMogli** [apache/incubator-mxnet#8298]
Can you add the pre-trained model to dropbox please? It is difficult to obtain the pre-trained model from baidu from outside of china.

**Comment by szha** [comment]
which models are you referring to?

**Comment by szha** [comment]
@apache/mxnet-committers: This issue has been inactive for the past 90 days. It has no label and needs triage.

For general "how-to" questions, our [user forum](https://discuss.mxnet.io/) (and [Chinese version](https://discuss.gluon.ai/)) is a good place to get help.
